internal/util/k8sutil/api.go

```diff
@@ -112,6 +112,9 @@ func GetCustomResourceDefinitions(crdsDir string) (
 func DefinitionsForV1CustomResourceDefinitions(crds ...apiextv1.CustomResourceDefinition) (keys []registry.DefinitionKey) {
 	for _, crd := range crds {
 		for _, ver := range crd.Spec.Versions {
+			if ver.Deprecated {
```

Comment:
Deprecated CRD versions are still permitted. I think the request was to ignore unserved versions, since these cannot be created at all.